Sources/FluentKit/Query/QueryBuilder+Paginate.swift

```diff
@@ -8,8 +8,8 @@ extension QueryBuilder {
     public func paginate(
         _ request: PageRequest
     ) -> EventLoopFuture<Page<Model>> {
-        let count = self.count()
-        let items = self.copy().range(request.start..<request.end).all()
+        let count = self.copy().count()
+        let items = self.range(request.start..<request.end).all()
```

Comment:
This change should be reverted, we need `self.copy().range(...)`, and `count()` already copies internally.

Reply:
Side note, we could write a test for this by doing a paginate followed by an `all()` on the same builder.